src/elements/GeoWizardLayout/GeoWizardLayout.vue

```diff
@@ -21,7 +21,7 @@
       </section>
 
       <footer
-        v-if="hasFooter"
+        v-if="!!this.$slots.footerTrailing || !!this.$slots.footerLeading"
```

Comment:
```suggestion
        v-if="!!$slots.footerTrailing || !!$slots.footerLeading"
```
Should be without `this`, right?

Reply:
True! 🤦‍♂ Fixed now!